torch/ao/quantization/utils.py

```diff
@@ -360,3 +361,26 @@ def _parent_name(target):
         return '', r[0]
     else:
         return r[0], r[1]
+
+def nonparam_type(module):
+    """
+    Returns type(module) or the original
+    type if module is currently parametrized
+    """
+    if is_parametrized(module):
+        return module.__class__.__bases__[0]
+    else:
+        return type(module)
+
+def is_leaf(module):
```

Comment:
`is_leaf` is already used in the quantization codebase for the FX use case (a leaf is a module not traced through), could we pick a different name for this?

Reply:
yeah, I can do something like parametrized_check_for_leaf or something like that?

Reply:
i changed it to is_leaf_or_only_parametrized

Reply:
how about something like `has_no_children_ignoring_parametrizations`?

Reply:
ok